closures.md

```diff
@@ -167,6 +168,76 @@ The implementation are the private variables `name`, `age` and the code inside t
 
```

Comment:
Word, I'll touch it up. Not going to cover this section in the lesson anymore but will clean it up if students want to reference.